.github/workflows/linux-dispatch-build-bottle.yml

```diff
@@ -0,0 +1,145 @@
+name: Dispatch build bottle (Linux)
+
+on:
+  workflow_dispatch:
+    inputs:
+      formula:
+        description: Formula name
+        required: true
+      issue:
+        description: Issue number, where comment on failure would be posted
+        required: false
+      upload:
+        description: Whether to upload built bottles to Bintray or not
+        required: false
+
+env:
+  HOMEBREW_DEVELOPER: 1
+  HOMEBREW_GITHUB_ACTIONS: 1
+  HOMEBREW_NO_AUTO_UPDATE: 1
+
+jobs:
+  bottle:
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/ubuntu16.04:master
+    env:
```

Comment:
I believe those 2 HOMEBREW_GIT env vars are unneeded.

Reply:
A lot of tests expect a git user and email to be set. Or we can set these for each test manually, but it was convenient to have these here.

Reply:
I believe those are set in test-bot already.

Reply:
If not, then Homebrew/actions/git-user-config does it fit sure

Reply:
I removed them, and we will have a look in case any build fails due to that.